recipes/vvenc/all/conanfile.py

```diff
@@ -0,0 +1,73 @@
+import os
+from conan.tools.build import stdcpp_library
+from conan import ConanFile, conan_version
+from conan.tools.cmake import CMakeToolchain, CMake, cmake_layout, CMakeDeps
+from conan.tools.files import get, copy, rmdir
+from conan.errors import ConanInvalidConfiguration
+
+
+class vvencRecipe(ConanFile):
+    name = "vvenc"
+    license = "BSD-3-Clause"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "Fraunhofer Versatile Video Encoder (VVenC)"
+    topics = ("video", "encoder", "codec", "vvc", "h266")
+    homepage = "https://www.hhi.fraunhofer.de/en/departments/vca/technologies-and-solutions/h266-vvc.html"
+    package_type = "library"
+
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+
+    def validate(self):
+        if conan_version.major < 2:
+            raise ConanInvalidConfiguration("conan v1 is not supported")
+
```

Comment:
```suggestion
```
Please, does not enforce it, this recipe is simple enough to support both major versions.

Reply:
Conan v1 is still mandatory in CCI, please, remove it.

Reply:
@prince-chrismc @jcar87 is it possible to relax a requirement for new recipes (btw, I don't see that requirement in doc, doc just says `Make sure you are using a recent [Conan client](https://conan.io/downloads) version`?
it doesn't make much sense to waste time and effort for something that is already legacy for several moths.
I suppose we can safely assume new recipe can worry only about an actual conan version going forward.
as recipe didn't exist for v1 ever, it cannot break anyone, right?